File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag41e42/ExercicioE.java
package manzanoPag41e42;

import java.util.Scanner;

import java.lang.Math;

public class ExercicioE {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		float a = 0, b = 0, c = 0, x = 0, x2 = 0, delta = 0, raiz = 0;

		System.out.println("Considere a formula: ax^2+bx+c=0");
		System.out.println("Insira o valor de a:");
		a = sc.nextFloat();

		if (a != 0) {
			System.out.println("Insira o valor de b:");
			b = sc.nextFloat();
			System.out.println("Insira o valor de c:");
			c = sc.nextFloat();
			delta = (float) Math.pow(b, 2) - 4 * a * c;
			if (delta < 0) {
				System.out.println("A equação não terá raizes reais");
			} else {
				raiz = (float) Math.sqrt(delta);
				x = ((-1 * b) + raiz) / (2 * a);
				x2 = ((-1 * b) - raiz) / (2 * a);
				if (x2 == x) {
					System.out.println("O resultado possivel de x é:" + x);
				} else {
					System.out.println("Os resultados possiveis de x são:" + x + " e " + x2);
				}
			}
		} else {
			System.out.println("Valor invalido. Por favor, reinicie o programa.");
		}

		sc.close();

	}

}

File: LogicaDeProgramacao(JavaScript)/Exercicios pag 41 e 42(Manzano)/js/exercicioE.js
/** e. Efetuar a leitura de três valores (variáveis A, B e C) e efetuar o cálculo da equação completa de
segundo grau, apresentando as duas raízes, se para os valores informados for possível efetuar o
referido cálculo. Lembre-se de que a variável A deve ser diferente de zero */

a = 0, b = 0, c = 0, x = 0, x2 = 0, delta = 0, raiz = 0


alert("Considere a formula: ax^2+bx+c=0")
   a = parseFloat(prompt("Insira o valor de a:"))

   if (a != 0) {
      b = parseFloat(prompt("Insira o valor de b:"))
      c = parseFloat(prompt("Insira o valor de c:"))
      delta = b^2 - 4*a*c
      if (delta < 0) {
         alert("A equação não terá raizes reais")
      } else {
         raiz =  Math.sqrt(delta);
         x = ((-1*b)+ raiz)/(2*a)
         x2 = ((-1*b)- raiz)/(2*a)
         if (x2 == x) {
            alert("O resultado possivel de x é:"+x)
         } else {
            alert("Os resultados possiveis de x são:"+x+ " e "+ x2)
         }
      }
   } else {
      alert ("Valor invalido. Por favor, reinicie o programa.")
   }